gem/Gemfile

```diff
@@ -16,8 +16,3 @@
 source 'https://rubygems.org'
 
 gemspec
-
-gem 'coveralls', require: false
-gem 'rspec'
-gem 'rubocop'
-gem 'thor' # enable: bundle exec thor spec
```

Comment:
it should be covered from `gemspec` in the line above. what did it give you when you do `bundle list`, here is mine:

```
Gems included by the bundle:
  * CFPropertyList (2.3.5)
  * activesupport (4.2.8)
  * ast (2.3.0)
  * bundler (1.14.6)
  * claide (1.0.1)
  * coderay (1.1.1)
  * colored (1.2)
  * coveralls (0.8.19)
  * diff-lcs (1.3)
  * docile (1.1.5)
  * earlgrey (0.2.0)
  * i18n (0.8.1)
  * json (2.0.3)
  * method_source (0.8.2)
  * minitest (5.10.1)
  * nanaimo (0.2.3)
  * parser (2.4.0.0)
  * powerpack (0.1.1)
  * pry (0.10.4)
  * rainbow (2.2.1)
  * rake (11.3.0)
  * rspec (3.5.0)
  * rspec-core (3.5.4)
  * rspec-expectations (3.5.0)
  * rspec-mocks (3.5.0)
  * rspec-support (3.5.0)
  * rubocop (0.47.1)
  * ruby-progressbar (1.8.1)
  * simplecov (0.12.0)
  * simplecov-html (0.10.0)
  * slop (3.6.0)
  * term-ansicolor (1.4.0)
  * thor (0.19.4)
  * thread_safe (0.3.6)
  * tins (1.13.2)
  * tzinfo (1.2.2)
  * unicode-display_width (1.1.3)
  * xcodeproj (1.4.2)
```